### PR TITLE
Update README with updated coprs

### PR DESCRIPTION
**Pull request by inknos**
Doc updates relates to fedora version being released soon and to these changes in our nightly builds:
https://github.com/rpm-software-management/ci-dnf-stack/pull/1396
https://github.com/rpm-software-management/ci-dnf-stack/pull/1394

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,15 +37,30 @@ Note, however, that DNF5 cannot yet interface with the following programming lan
...
+DNF5 is available since Fedora 38. To install the DNF5 package manager, use either of the following commands, depending on your Fedora release:
 
 | Fedora release | Command |
 | --------------------------- | ------- |
-| 36 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
-| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
+| 36 | `sudo dnf copr enable rpmsoftwaremanagement/dnf-nightly && sudo dnf install dnf5`[^copr] |
+| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-nightly && sudo dnf install dnf5`[^copr] |
 | 38                 | `sudo dnf install dnf5` |
+| 39                 | `sudo dnf install dnf5` |
```
Comment (ppisar) [inline README.md:47]: I would remove a line for Fedora 36 and added a line for Fedora 40.

Comment (inknos) [reply README.md:47]: oh, right, I will change that

Comment (inknos) [reply README.md]: I actually changed it to `38 or newer`, what do you think?

File: README.md [hunk 2]
```diff
@@ -37,15 +37,28 @@ Note, however, that DNF5 cannot yet interface with the following programming lan
...
 
-To install the DNF5 package manager, use either of the following commands, depending on your Fedora release:
+DNF5 is available since Fedora 38. To install the DNF5 package manager, use either of the following commands, depending on your Fedora release:
 
 | Fedora release | Command |
 | --------------------------- | ------- |
-| 36 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
-| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
-| 38                 | `sudo dnf install dnf5` |
+| 37 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-nightly && sudo dnf install dnf5`[^copr] |
```
Comment (evan-goode) [inline README.md:44]: Should this be `rpmsoftwaremanagement/dnf-nightly`?

**Review by evan-goode — CHANGES_REQUESTED** [review]
Looks good overall, I suggest writing a README/description for https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/dnf5-testing-nightly before merging.

**Comment by evan-goode** [comment]
A caveat to recommending F37 users install DNF 5 via the dnf-nightly Copr: the next time they `dnf upgrade`, they'll upgrade to the nightly builds of DNF 4 also, which may be unexpected.

File: README.md (hunk 2 above)
Comment (ppisar) [reply README.md:44]: Also the [^copr] foot note points to <https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/dnf5-unstable/>. I think it should point to the same COPR repository as the command highlights.

**Comment by inknos** [comment]
> Looks good overall, I suggest writing a README/description for https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/dnf5-testing-nightly before merging.

I updated the description in copr. @evan-goode , do you think we should also provide a description upstream?

**Comment by evan-goode** [comment]
> > Looks good overall, I suggest writing a README/description for https://copr.fedorainfracloud.org/coprs/rpmsoftwaremanagement/dnf5-testing-nightly before merging.
> 
> I updated the description in copr. @evan-goode , do you think we should also provide a description upstream?

A description in Copr is what I meant, LGTM!

**Review by evan-goode — APPROVED** [review]

**Pull request merged**